### PR TITLE
ZEVA 212: Added checkbox to allow users to claim additional credit 

### PR DESCRIPTION
**Pull request by amichard**
Changelog:
- Added checkbox to claim additional credit for certain engine types
- Updated model so that it can save the claim for additional credit
- Checking additional credit shows upload files in the model creation form
- Added new calclulation

### CODE REVIEW

File: backend/api/models/vehicle.py [hunk 1]
```diff
@@ -66,6 +66,11 @@ class Vehicle(Auditable):
         on_delete=models.PROTECT,
         null=True
     )
+    additional_credit = models.BooleanField(
```
Comment (klefler) [inline backend/api/models/vehicle.py:69]: Just wondering if the column name should be consistent with other boolean columns in the database. I think the convention used was 'is_' or 'has_' ... would 'has_additional_credit' make sense?

File: backend/api/models/vehicle.py [hunk 2]
```diff
@@ -66,6 +66,11 @@ class Vehicle(Auditable):
...
+        default=False,
+        db_comment="Boolean Field to see whether the vehicle should get "
+                   "additional credit"
+    )
```
Comment (klefler) [inline backend/api/models/vehicle.py:73]: If this column is specific to only the US06 credit request, could this be added to the column comment?

i.e. "Boolean field used to claim whether the vehicle should get additional credit for passing the US06 range test."

File: backend/api/models/vehicle.py (hunk 1 above)
Comment (amichard) [reply backend/api/models/vehicle.py:69]: initially tried that, but couldn't change "Can claim additional credit" into an "is/has" format

File: backend/api/models/vehicle.py (hunk 2 above)
Comment (amichard) [reply backend/api/models/vehicle.py:73]: il change the comment

File: backend/api/models/vehicle.py (hunk 1 above)
Comment (amichard) [reply backend/api/models/vehicle.py:69]: maybe "has_passed_us06_test" might make more sense

Comment (klefler) [reply backend/api/models/vehicle.py:69]: ya that or "has_claimed_us06_credit"?

**Pull request merged**